### ISSUE TITLE
Remove a filter for a particular column

### ISSUE THREAD

**Issue by glenpadua** [DevExpress/devextreme-reactive#777]
Hi I'd like to disable the filter for a particular column. From the docs I noticed that the FilteringState plugin's `changeColumnFilter` - `Adds, changes or removes a filter. Pass null to the config argument to remove the filter associated with the specified column.` can be used.

But I'm not really sure how an "Action" should be used. Any help will be much appreciated!

**Comment by kvet** [comment]
We've introduced such a functionality in [1.1.0-beta.1](https://github.com/DevExpress/devextreme-reactive/releases/tag/v1.1.0-beta.1). It's available through the `FilteringState` plugin's `columnExtensions` property:
 
```js
<FilteringState
  columnExtensions={[
    { columnName: 'name', filteringEnabled: false },
  ]}
/>
```
 
Moreover, other Grid features such as editing, filtering, grouping, and column visibility changing can be configured in a similar way. The state management plugins (`EditingState`, `FilteringState`, `GroupingState`, `TableColumnVisibility`) contain the related `columnExtensions` properties.
 
You can install React Grid v1.1.0-beta.1 with the `next` tag:

`npm i --save @devexpress/dx-react-core@next @devexpress/dx-react-grid@next`
`npm i --save @devexpress/dx-react-grid-bootstrap3@next`
`npm i --save @devexpress/dx-react-grid-material-ui@next`

Please check this out and send us your feedback if any!

**Comment by kvet** [comment]
The described functionality is available in React Grid starting with [v1.1.0](https://github.com/DevExpress/devextreme-reactive/releases/tag/v1.1.0) that is officially released.

**Comment by MaximKudriavtsev** [comment]
Hi @efstathiosntonas,

As far as I understand, you want to clear a filter if its value is 'All'. If so, please look at this [sample](https://codesandbox.io/s/j26q902wm3).
